lib/Twig/Cache/Filesystem.php

```diff
@@ -46,7 +46,7 @@ public function generateKey($name, $className)
      */
     public function load($key)
     {
-        @include_once $key;
+        if (file_exists($key)) include_once $key;
```

Comment:
Ok, so, I'm a javascript developer, I don't have @, so have a good day, I need to go find a if(exists)openfile substitute for all my projects.

Meanwhile I will begin with the safe and wellcoded try{ openfile }catch(e){}, but don't worry about unknown catched errors, they never happens.